### ISSUE TITLE
Missing optimizer and scheduler

### ISSUE THREAD

**Issue by liuxz0801** [modelscope/modelscope#58]
I run  some examples from docs . but i found some optimizers and schedulers missed which is used in configrations .  such as "AdamW". in official docs ,it may casued by that my modelscope is too low .but  my version is 1.1.0. it is nearly most new one.   if  i want to use "AdamW" , Should  I  code  and register it by my self ?

**Comment by yingdachen** [comment]
can you please specify which model you are trying to use? a code snippet that may reproduce the problem may be the best.

**Comment by liuxz0801** [comment]
well, i just finetuned gpt3_base. 
when i run examples in https://modelscope.cn/models/damo/nlp_gpt3_text-generation_chinese-large/summary :
    cfg.train.optimizer = {
        "type": "AdamW",
        "lr": 3e-4
    }


i got the warning:

modelscope - WARNING - ('OPTIMIZER', 'default', 'AdamW') not found in ast index file
modelscope - WARNING - ('LR_SCHEDULER', 'default', 'StepLR') not found in ast index file

then i check metainfo.py, i can not find  adamw in it.

**Comment by zzclynn** [comment]
> 
Please ignore these warnings now, we register those components from torch during compling stage, while without handling those warnings.

If you want to use your own components, please refer https://github.com/modelscope/modelscope/blob/master/modelscope/trainers/optimizer/child_tuning_adamw_optimizer.py this method.
